types/scratch-env/index.d.ts

```diff
@@ -0,0 +1,199 @@
+// Type definitions for scratch-env 3.0
```

Comment:
```suggestion
// Type definitions for scratch-vm (extension environment) 3.0
```

to make it clear this doesn’t correspond to an npm package called `scratch-env`. webpack-env says

```js
// Type definitions for webpack (module API) 1.15
```